### PR TITLE
00020_comments

### PR DESCRIPTION
**Pull request by zdenkas**

### CODE REVIEW

File: notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb [hunk 1]
```diff
@@ -1224,7 +1224,11 @@
    "id": "162968cb-3f83-4aab-8c76-e557e5a21cc0",
```
Comment (zdenkas) [inline notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1224]: <p>@dianagroch can you please check this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/16/discussion/#comment-1487443782'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='31'/>

Comment (zdenkas) [inline notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1224]: <p>@dianagroch can you please check this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/16/discussion/#comment-1487443787'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='36'/>

Comment (zdenkas) [inline notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1224]: <p>@dianagroch can you please check this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/16/discussion/#comment-1487443788'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='42'/>

Comment (dianagroch) [reply notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1224]: <h4>hg38 X:154412187 - is three time in dataset (rows 57, 58 and 65)</h4><h4><br></h4><p>for rows 57 and 58:</p><h4>in position hg38 X:154412187 is T (checked at UCSC and NCBI)</h4><p>so &nbsp;&nbsp;c.211T&gt;C should be correct </p><p> </p><p>for row 65: </p><p>does not fit conversion from hg37 should be hg38 <a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chrX:154412202-154412205" target="_blank">chrX:154412203</a> (not 154412187)</p><p>should be C in this position</p><p>than this is correct: NM_000116.5(TAFAZZIN):c.227C&gt;G (p.Pro76Arg)</p><p> </p><p> </p><p> </p><p> </p><p> </p>

Comment (dianagroch) [reply notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1224]: <p>if the correct is c.590G&gt;T (according to paper) than the correct position should be:</p><p>hg38 <a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chrX:154420037-154420040" target="_blank">chrX:154420038</a> p.G197V</p><p> </p><p>question is if the ref. 23 is correct in this case (see row 256 for same variant, in ref 22 patient 4 - different mutation G877T )</p><p>this should be checked with submitters</p>

Comment (dianagroch) [reply notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb:1224]: <p><strong>row 273:</strong></p><p><strong>Lift over: hg19:</strong><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;position=chrX:153648426-153648429" target="_blank">X:153648427</a> <strong>hg38:</strong><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chrX:154420087-154420090" target="_blank">chrX:154420088</a></p><p>should be C in this position , c.640C&gt; T, p.H214Y - correct</p><p> </p><p>yes, just problem with conversion</p>

**Pull request merged**